tests/lib/location/prj.conf

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_UNITY=y
+CONFIG_RING_BUFFER=n
+CONFIG_ASSERT=y
+CONFIG_HEAP_MEM_POOL_SIZE=5120
+
+CONFIG_LOCATION=y
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_SMS=y
+
+# TODO: We should use nRF Cloud but maybe after it has converted to use rest client
```

Comment:
Check the comment, nRF Cloud has already been updated to use REST client, right?

Reply:
Yep, I'll still use HERE because using nRF Cloud libraries is a bigger task due to all dependencies. So not sure whether to mock nrf_cloud APIs or rest_client